src/particle/pdgid/__init__.py

```diff
@@ -68,13 +68,17 @@
 from .functions import is_valid
 from .functions import abspid
 
+from .functions import is_quark
 from .functions import is_lepton
 from .functions import is_hadron
 from .functions import is_meson
 from .functions import is_baryon
+from .functions import is_heavy_flavor
```

Comment:
Is "heavy flavor" really unambiguous? Does it include only c and b or also s? And what about top? If it is not completely obvious, then I would not expose this. Better  teach people to select based on the quark content.

Reply:
Yes it is unambiguous - c,b,t. If we wanted to be totally strict I would say that nobody is 100% consistent in the sense that one should also include fourth-generation q-qbar states, if they were to exist. This 4th generation is encoded in the PDG scheme but not considered in hadron when saying heavy flavour hadrons, etc.

So yes, one can do without this definition, though the definition from the PDG doc is clear about what it means (and I made sure the docs provide the necessary information to the best of my knowledge).